app/src/main/java/com/duckduckgo/app/cta/ui/CtaViewModel.kt

```diff
@@ -320,7 +322,13 @@ class CtaViewModel @Inject constructor(
             }
 
             if (areInContextDaxDialogsCompleted()) {
-                return if (brokenSitePrompt.shouldShowBrokenSitePrompt(nonNullSite.url)) {
+                val detectedRefreshPatterns = brokenSitePrompt.getUserRefreshesCount()
+                refreshPixelSender.onRefreshPatternDetected(detectedRefreshPatterns)
+
+                return if (3 in detectedRefreshPatterns &&
```

Comment:
If we decide to not query the refreshes in `shouldShowBrokenSitePrompt`, which might make sense because of the pixel, I think we should still send the number of refreshes as an attribute, and keep all the logic within the method